articles/libraries/lock/v10/index.md

```diff
@@ -114,7 +113,7 @@ function showLoggedIn() {
 This example demonstrates using Lock 10 with a Single Page Application (SPA). To learn how Lock can be modified to provide frictionless authentication for any app, see the [API Reference][lock-api] and the [Configuration Options Reference][lock-customization]. For details specifically about customizing the look and feel of Lock in your app, please take a look at the [UI Customization][ui-customization] page.
```

Comment:
This content does not look like it should be a note. You should move this to a `Keep Reading` paragraph at the end. Use the `next-steps` element.